storybook/config.js

```diff
@@ -22,28 +24,34 @@ global.___loader = {
   hovering: () => {},
 }
 // Gatsby internal mocking to prevent unnecessary errors in storybook testing environment
-global.__PATH_PREFIX__ = ""
-  // This is to utilized to override the window.___navigate method Gatsby defines and uses to report what path a Link would be taking us to if it wasn't inside a storybook
-  window.___navigate = pathname => {
-  action("NavigateTo:")(pathname)
+global.__PATH_PREFIX__ = ''
+// This is to utilized to override the window.___navigate method Gatsby defines and uses to report what path a Link would be taking us to if it wasn't inside a storybook
+window.___navigate = pathname => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected dangling '_' in '___navigate'.](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4765408)